atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/RowOrDynamicColumnRenderer.java

```diff
@@ -186,15 +171,22 @@ private void bytesHydrator() {
                     }
                     line("__index += ", comp.getType().getHydrateSizeCode(var), ";");
                 }
-                line("return of(", Joiner.on(", ").join(vars), ");");
+                line("return new ", Name, "(", Joiner.on(", ").join(vars), ");");
```

Comment:
per above, I guess this should be

    line("return ", Name, ".of(", Joiner.on(", ").join(vars), ");");